src/templates/floatingPlayer.js

```diff
@@ -0,0 +1,15 @@
+export default (id, ariaLabel = '') => {
```

Comment:
Instead of defining a new template, re-use the player one and add a third argument, `hasFloatOnScroll` and conditionally wrap the template in a `<div>` container.